dask/dataframe/core.py

```diff
@@ -4461,9 +4461,12 @@ def hash_shard(df, nparts, split_out_setup=None, split_out_setup_kwargs=None):
         h = split_out_setup(df, **(split_out_setup_kwargs or {}))
     else:
         h = df
-    h = hash_pandas_object(h, index=False)
+    from .utils import hash_object_dispatch
```

Comment:
Can this import be done at the top of the file?

Reply:
Yes, moved it to the top